docs/rules/order.md

```diff
@@ -385,6 +385,8 @@ Valid properties and their values include:
 
  - **`caseInsensitive`**: use `true` to ignore case and `false` to consider case when sorting
 
+ - **`orderByFullPathString`**: use `false` to split by paths and sort by each one and `true` to sort by the full un-split path string
```

Comment:
is there a better way to phrase this?

like, perhaps one way sorts the way certain operating systems and IDEs do, and the other sorts the way other ones do? (with an example list of each)